drud/ddev
# 20190911 makensis homebrew

## The Problem/Issue/Bug:

The upstream homebrew makensis was broken for a while, see https://github.com/drud/ddev/issues/1773; it was temporarily fixed in https://github.com/drud/ddev/pull/1783

However, we created a fix PR in homebrew queue, https://github.com/Homebrew/homebrew-core/pull/43558, and it resolved this. So now we can go back to the upstream homebrew version of makensis. 

## How this PR Solves The Problem:

Remove the temporary makensis brew recipe used in #1783 

## Manual Testing Instructions:

* Download the built windows installer from a *release build* (which uses macOS). 
* Run the installer and make sure that it installs lots of things at the last step. 

## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:
